modulefiles/build_gaea_intel.lua

```diff
@@ -20,7 +20,6 @@ local MKLROOT="/opt/intel/oneapi/mkl/2022.0.2/"
 prepend_path("LD_LIBRARY_PATH",pathJoin(MKLROOT,"lib/intel64"))
 pushenv("MKLROOT", MKLROOT)
 
```

Comment:
@christinaholtNOAA , 
all these could be removed, too, as they were needed for GSI only:
```
prepend_path("LD_LIBRARY_PATH",pathJoin(MKLROOT,"lib/intel64"))
pushenv("MKLROOT", MKLROOT)
pushenv("CRAYPE_LINK_TYPE","dynamic")
```

Reply:
Alternatively, I could take care of these when updating the modulefiles #889 , as it may be scheduled to be merged after the present one (993)

Reply:
@natalie-perlin I can go ahead and remove those here.